### ISSUE TITLE
glossterm usage context clarity

### ISSUE THREAD

**Issue by GoogleCodeExporter** [w3c/epub-specs#526]
```
Another case of the definition being too wordy:

When the element on which the property appears is a dt element within a dl 
element marked with the glossary property, then the glossterm property on the 
given dt element is implied.

Why would the property appear if the property is implied?
```

Original issue reported on code.google.com by `mgarrish` on 14 Mar 2015 at 3:27

**Comment by mattgarrish** [comment]
This issue may be obsoleted depending on whether implied contexts are removed.

**Comment by mattgarrish** [comment]
Need to review removing as part of the ARIA harmonization.

**Comment by TzviyaSiegman** [comment]
DPUB-ARIA's definition of "glossary" (https://www.w3.org/TR/dpub-aria-1.0/#doc-glossary) requires that all children are `<dt>`, `<dd>`, or have the ARIA equivalents. Current SSV definitions allow for "glossary"  to be applied to any sectioning content. The wordy usage context is relevant only when glossary is applied to `<dl>`. Our options are to change requirements in SSV (restrict to `<dl>` and relevant children) or clarify the language for HTML context.
Recommended wording: Glossterm should be used only when the element to which it is applied is NOT a `<dt>` child of `<dl epub:type="glossary">`, in which case it is implicit.

**Comment by TzviyaSiegman** [comment]
Recommendation: Change usage context for glossterm to: Glossterm should be used only when the element to which it is applied is NOT a `<dt>` child of `<dl epub:type="glossary">`, in which case it is implicit.

**Comment by mattgarrish** [comment]
Only change I'd recommend is that we not say it's a "should", as that will lead to warnings. No reason an implied semantic can't be specified. Even ARIA in HTML can only note it's a waste of time to do, but what someone does with their time is their choice. We can probably just leave as:

The glossterm property is implied on dt elements within a dl element marked with the glossary property.

**Comment by mattgarrish** [comment]
Term has been updated as above:
http://rawgit.com/IDPF/epub-revision/master/build/epub/vocab/structure/epub-vocab-structure.html#glossterm

Please close this issue if no other changes are necessary.
